process/mock/poolsHolderFake.go

```diff
@@ -11,7 +11,8 @@ import (
 
 type PoolsHolderFake struct {
```

Comment:
Why is this called "Fake"? Can we change it to Mock?

Reply:
done